backend/canisters/user_index/api/src/updates/c2c_notify_chit.rs

```diff
@@ -0,0 +1,15 @@
+use serde::{Deserialize, Serialize};
+use types::TimestampMillis;
+
+#[derive(Serialize, Deserialize, Debug)]
+pub struct Args {
+    pub chit_balance: i32,
```

Comment:
I think this needs a timestamp, because they may not arrive in order, so the user index needs to be able to ignore values that aren't the latest.